templates/questions/show.html.twig

```diff
@@ -46,11 +45,12 @@
                                             {{ questionTag.tag }}
                                             </a>
                                         {% endfor %}
-                                    </div>
-                                    {% if question.askedAt == null %}
-                                        <button class="mt-5 btn btn-primary">Publish</button>
-                                    {% endif %}
-                                </form>                                
+                                </div>
+                                {% if question.askedAt == null %}
+                                <form action="{{ path('app_question_publish', { slug: question.slug }) }}" method="POST">    
```

Comment:
After looking at this again, I don't think you need a form at all here. You should be able to just have a link to this action. The post request isn't actually doing anything.

Also, I can't tell for sure, but it looks like the formatting might be off here. Just double check the indentations.

Reply:
Checked formatting and removed form, replaced button with anchor tags